### PR TITLE
Tool enhancement: dr_curve

### PR DESCRIPTION
**Pull request by rmassei**
Enhancement of the tool [dr_curve](https://toolshed.g2.bx.psu.edu/):
- Now there is a more complete summary (i.e. model AIC)
- Improve the visualization with ggplot2 (replicates and confidence intervals are now show in the plot)
- Added an additional model (BC.5)

### CODE REVIEW

**Review by bernt-matthias — COMMENTED** [review]
Looks good, we only need to bump the tool version.

File: tools/tox_tools/dose_responses/dose_response.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <tool id="dr_curve" name="Dose Response Curve for Toxicological Risk Assessment" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="23.0">
     <description>for Toxicological Risk Assessment</description>
     <macros>
-        <token name="@TOOL_VERSION@">3.0.1</token>
+        <token name="@TOOL_VERSION@">4.0.1</token>
         <token name="@VERSION_SUFFIX@">2</token>
```
Comment (bernt-matthias) [inline tools/tox_tools/dose_responses/dose_response.xml:5]:
```suggestion
        <token name="@TOOL_VERSION@">4.0</token>
        <token name="@VERSION_SUFFIX@">0</token>
```

Comment (rmassei) [reply tools/tox_tools/dose_responses/dose_response.xml:5]: 😅

**Comment by bernt-matthias** [comment]
In order to fix the R stuff you can use this one: https://github.com/Helmholtz-UFZ/galaxy-tools/blob/main/.github/styler.R .. just need to install `styler`.

File: tools/tox_tools/dose_responses/dose_response.xml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <tool id="dr_curve" name="Dose Response Curve for Toxicological Risk Assessment" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="23.0">
     <description>for Toxicological Risk Assessment</description>
     <macros>
-        <token name="@TOOL_VERSION@">3.0.1</token>
-        <token name="@VERSION_SUFFIX@">2</token>
+        <token name="@TOOL_VERSION@">4.0</token>
```
Comment (bernt-matthias) [inline tools/tox_tools/dose_responses/dose_response.xml:4]:
```suggestion
        <token name="@TOOL_VERSION@">3.0.1</token>
        <token name="@VERSION_SUFFIX@">3</token>
```

File: tools/tox_tools/dose_responses/dose_response.xml [hunk 3]
```diff
@@ -2,7 +2,8 @@
     <description>for Toxicological Risk Assessment</description>
     <macros>
         <token name="@TOOL_VERSION@">3.0.1</token>
-        <token name="@VERSION_SUFFIX@">2</token>
+        <token name="@VERSION_SUFFIX@">3</token>
+        <token name="@VERSION_SUFFIX@">0</token>
```
Comment (bernt-matthias) [inline tools/tox_tools/dose_responses/dose_response.xml:6]:
```suggestion
        <token name="@VERSION_SUFFIX@">3</token>
```

**Comment by bernt-matthias** [comment]
There is now a diff with the test-data. 

```
Output plot_output:  different than expected, difference (using diff):
( /tmp/tmpe3ivt70iimage_output.jpg v. /tmp/tmp25eq3tp4image_output.jpg )
Binary data detected, not displaying diff
```

I reverted the version token because it is also used in the requirement.

**Comment by rmassei** [comment]
Good to go!

**Pull request merged**